dakota/oat/cooling-time/README.md

```diff
@@ -0,0 +1,24 @@
+# Hello. Welcome to the One-at-a-time Cooling time sensitivity analysis study. 
+To run the simulation, type this into the command line: 
+
+`dakota -i cooling-time.in -o cooling-time.out`
+
+This will generate 5 sqlite output files in the 
+`../../../cyclus-files/oat/cooling-time/` directory. 
```

Comment:
Why is the directory called "cyclus-files" ? Is there a more meaningful way to describe the contents of this directory?